docs/api/config.md

```diff
@@ -254,7 +256,7 @@ In addition to the JSON defined on the `tauri.conf.json` file, Tauri reads a pla
     { property: "resizable", optional: true, type: "boolean", description: `Whether the window is resizable or not.` },
     { property: "skipTaskbar", optional: true, type: "boolean", description: `Whether or not the window icon should be added to the taskbar.` },
     { property: "title", optional: true, type: "string", description: `The window title.` },
-    { property: "transparent", optional: true, type: "boolean", description: `Whether the window is transparent or not. Note that on \`macOS\` this requires the \`macos-private-api\` feature flag, enabled under \`tauri.conf.json > tauri > macOSPrivateApi\`. WARNING: Using private APIs on \`macOS\` prevents your application from being accepted for the \`App Store\`.` },
+    { property: "transparent", optional: true, type: "boolean", description: `Whether the window is transparent or not. Note that on \`macOS\` this requires the \`macos-private-api\` feature flag, enabled under \`tauri.conf.json > tauri > macosPrivateApi\`. WARNING: Using private APIs on \`macOS\` prevents your application from being accepted for the \`App Store\`.` },
```

Comment:
Needs to be fixed in tauri. Adding the comment so I don't forget it...

Reply:
Wdym exactly?

Reply:
It's macOSPrivateApi not macosPrivateApi but this file is generated from the schema.json file over at tauri